source/gui/settingsDialogs.py

```diff
@@ -3274,27 +3274,83 @@ def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 		index = [x.value for x in AddonsAutomaticUpdate].index(config.conf["addonStore"]["automaticUpdates"])
 		self.automaticUpdatesComboBox.SetSelection(index)
 
-		# Translators: This is the label for a text box in the add-on store settings dialog.
-		self.addonMetadataMirrorLabelText = _("Server &mirror URL")
-		self.addonMetadataMirrorTextbox = sHelper.addLabeledControl(
-			self.addonMetadataMirrorLabelText,
-			wx.TextCtrl,
+		# Translators: The label for the server  mirror on the Add-on store Settings panel.
+		mirrorBoxSizer = wx.StaticBoxSizer(wx.HORIZONTAL, self, label=_("Server mirror"))
```

Comment:
Pedantic/subjective but maybe this is more accurate phrasing right? Making the subject the server/URL, not the mirror

```suggestion
		mirrorBoxSizer = wx.StaticBoxSizer(wx.HORIZONTAL, self, label=_("Mirror server"))
```

Reply:
Hmm, I don't think so, as this allows one to specify the URL to an alternative Add-on Store server. While I suppose this is in itself a server, I think "server mirror" is fine.

Reply:
@SaschaCowley for what it's worth, I agree with @seanbudd. Other than just "sounding backwards", AFAIK, in general usage for mirrors, the URL is to the mirror. Rarely does somebody mirror an entire server. The context is usually that the URL points to the mirror of the data, not necessarily a mirror of a server. 
So calling it "mirror server", means it is the thing that serves/makes available, the mirror.

I know in other places you write "update server mirror", implying that the mirror is of the update server. That sounds less weird than here, because there the word "server" can be associated with "update", and "mirror" applies to the phrase rather than the single noun. Whereas here it is just "Server mirror".

Maybe dodge the whole question by calling it "Mirror address" or something similar?

Reply:
Another option could be changing it to "Add-n Store mirror", "Store mirror", or just "Mirror". Happy to go with whatever folks think is most appropriate.

Reply:
My preference is still "Mirror server". I feel like add-on store is contextual to the panel, and "Mirror" is somewhat unclear on it's own. I think "Mirror host" might be better than "Mirror address" but similar goals.